### PR TITLE
[ISSUE #278]Translation for "Deployment-部署方式" Section in the v4.x & 5.0 Document (CN -> EN)

### LINKED ISSUE #278: Translation for "Deployment-部署方式" Section in the v4.x & 5.0 Document (CN -> EN)
The Chinese version of the "Deployment-部署方式" section in the v4.x & v5.0 document has now been finished. However, its English version is still incomplete. 

We now require someone who knows both Chinese and English and understands the technical terms of Apache RocketMQ to build it. The translation work is not only copy/paste from translation tools but also requires you to have some basic knowledge of Apache RocketMQ and technical writing.

### PR DESCRIPTION
**Pull request by mxsm**
Please do not create a Pull Request without creating an issue first. 

## What is the purpose of the change

close #278 

## Brief changelog

- Translation for "Deployment-部署方式" Section in the v4.x & 5.0 Document (CN -> EN)

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [ ] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: i18n/en/docusaurus-plugin-content-docs/current/04-deployment/15deploy.md [hunk 1]
```diff
@@ -1,143 +1,140 @@
...
+# Deployment method
 
-<!-- ## 集群搭建 -->
-
-## 单Master模式
+## Single Master mode
 
 :::caution
-这种方式风险较大，一旦Broker重启或者宕机时，会导致整个服务不可用。不建议线上环境使用, 可以用于本地测试。
+This mode carries a higher risk, as a restart or failure of the Broker will result in the entire service being unavailable. It is not recommended for use in online environments, but can be used for local testing.
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/current/04-deployment/15deploy.md]: It is not recommended for use in online environments, but can be used for local testing.

---

It is not recommended in online environments, but can be used for local testing.

File: i18n/en/docusaurus-plugin-content-docs/current/04-deployment/15deploy.md [hunk 2]
```diff
@@ -1,143 +1,140 @@
...
 The broker[broker-a, 192.169.1.2:10911] boot success...
 ```
 
-## 多Master模式
-
-一个集群无Slave，全是Master，例如2个Master或者3个Master，这种模式的优缺点如下：
+## Multiple master mode
 
-- 优点：配置简单，单个Master宕机或重启维护对应用无影响，在磁盘配置为RAID10时，即使机器宕机不可恢复情况下，由于RAID10磁盘非常可靠，消息也不会丢（异步刷盘丢失少量消息，同步刷盘一条不丢），性能最高；
+there are no slaves in the cluster and all are masters, such as 2 masters or 3 masters. The advantages and disadvantages of this mode are as follows:
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/current/04-deployment/15deploy.md]: there are no slaves in the cluster and all are masters, such as 2 masters or 3 masters.

->

The advantages and disadvantages of a cluster which is full of masters without slaves (e.g. 2 or 3 masters) is as follows:

**Review by tsunghanjacktsai — CHANGES_REQUESTED** [review]

File: i18n/en/docusaurus-plugin-content-docs/current/04-deployment/15deploy.md (hunk 1 above)
Comment (mxsm) [reply i18n/en/docusaurus-plugin-content-docs/current/04-deployment/15deploy.md]: @tsunghanjacktsai  I will re-submit pr and fix it

**Comment by mxsm** [comment]
Hi @tsunghanjacktsai  I hava resolved conflict and fixd that needs to change. PTAL~ thanks

**Review by tsunghanjacktsai — APPROVED** [review]

**Review by RongtongJin — APPROVED** [review]

**Pull request merged**